### ISSUE TITLE
[bug] Can't log in anymore with Fedilab and new beta

### ISSUE THREAD

**Issue by xxxserxxx** [superseriousbusiness/gotosocial#1479]
### Describe the bug with a clear and concise description of what the bug is.

Aftet upgrading to 0.7.0-rc2, I was getting a lot of errors with Fedilab on things that worked with 0.6.0, so I logged out of Fedilab, cleared my app cache, amd tried to log back in. Now I get "can't fetch token" errors, and can't log in with the app.

### What's your GoToSocial Version?

0.7.0-rc2

### GoToSocial Arch

amd64

### Browser version

Fedilab 3.15.2, Firefox 109.2.0 (Android)

### What happened?

I can't log in with Fedilab any longer.

It seems to succeed, but when it redirects back to the app, the app reports an error fetching the oauth token.

![Screenshot_20230211_210915_Firefox](https://user-images.githubusercontent.com/60757196/218290956-e652854f-d0e3-421a-8239-9f09aab78104.jpg)

![Screenshot_20230211_210927_Fedilab](https://user-images.githubusercontent.com/60757196/218290988-68f5dcbb-e6b0-4e8b-92d7-17ef7cf2b63a.jpg)

### What you expected to happen?

I expected to be able to log in.

### How to reproduce it?

1. Log out of an account
2. Attempt to log back in

### Anything else we need to know?

- The auth is through Firefox, and appears to be successful
- I can log in to /auth, in a fresh private tab (no cookies)
- I have logged out and back in to a different Mastodon.social account. For kicks, I also logged out and in to a Friendica account as well (Fedilab supports several ActivityPub servers). I haven't seen an auth failure with Fedilab on any other server
- I only noticed this with 0.7.0-rc2, but I haven't tried logging out and back in with Fedilab maybe ever before.
- I have no problems with 0.7.0-rc2 with tut (terminal client on Linux); I'm afraid to log out, now, but I'll try with a new tut config (same account) tomorrow.
- With 0.7.0 I'm seeing many issues with Fedilab: not being able to pull my own comments, profile issues, etc. I do not see these with tut (same GTS 0.7.0 account). 

It seems that Fedilab is not playing well with the new build, in general, but now that I can no longer log in with it, I'm unable to test.

**Comment by xxxserxxx** [comment]
In the interim, I completely uninstalled FediLab, and re-logged in to all of my other accounts (1 mastodon, 1 pixelfed, 1 Friendica). Only the ap.ser1.net is failing.

Here are the server logs from exactly when I try to log in with Fedilab:

```
Feb 13 09:02:28 tethys.ser1.net gotosocial[3800775]: timestamp="13/02/2023 09:02:28.778" func=middleware.Logger.func1.1 level=INFO latency=92.574881ms clientIP=xxx.xxx.44.10
userAgent="Mozilla/5.0 (Android 13; Mobile; rv:109.0) Gecko/109.0 Firefox/109.0" method=POST statusCode=302 path=/auth/sign_in msg="Found: wrote 23B"
Feb 13 09:02:29 tethys.ser1.net gotosocial[3800775]: timestamp="13/02/2023 09:02:29.227" func=middleware.Logger.func1.1 level=INFO latency=49.921922ms clientIP=xxx.xxx.44.10
userAgent="Mozilla/5.0 (Android 13; Mobile; rv:109.0) Gecko/109.0 Firefox/109.0" method=GET statusCode=200 path=/oauth/authorize msg="OK: wrote 908B"
Feb 13 09:02:29 tethys.ser1.net gotosocial[3800775]: timestamp="13/02/2023 09:02:29.946" func=middleware.Logger.func1.1 level=INFO latency=92.043µs clientIP=xxx.xxx.44.10 userAgent="Mozilla/5.0 (Android 13; Mobile; rv:109.0) Gecko/109.0 Firefox/109.0" method=GET statusCode=304 path=/assets/dist/_colors.css msg="Not Modified: wrote 0B"
Feb 13 09:02:29 tethys.ser1.net gotosocial[3800775]: timestamp="13/02/2023 09:02:29.950" func=middleware.Logger.func1.1 level=INFO latency=79.058µs clientIP=xxx.xxx.44.10 userAgent="Mozilla/5.0 (Android 13; Mobile; rv:109.0) Gecko/109.0 Firefox/109.0" method=GET statusCode=304 path=/assets/dist/base.css msg="Not Modified: wrote 0B"
Feb 13 09:02:32 tethys.ser1.net gotosocial[3800775]: timestamp="13/02/2023 09:02:32.289" func=middleware.Logger.func1.1 level=INFO latency=2.36789ms clientIP=xxx.xxx.44.10 userAgent="Mozilla/5.0 (Android 13; Mobile; rv:109.0) Gecko/109.0 Firefox/109.0" method=POST statusCode=302 path=/oauth/authorize msg="Found: wrote 23B"
Feb 13 09:02:33 tethys.ser1.net gotosocial[3800775]: timestamp="13/02/2023 09:02:33.404" func=middleware.Logger.func1.1 level=INFO latency=1.666646ms clientIP=xxx.xxx.44.10 userAgent=okhttp/4.9.2 method=POST statusCode=200 path=/oauth/token msg="OK: wrote 157B"
Feb 13 09:02:34 tethys.ser1.net gotosocial[3800775]: timestamp="13/02/2023 09:02:34.217" func=middleware.Logger.func1.1 level=INFO latency=2.941547ms clientIP=xxx.xxx.44.10 userAgent=okhttp/4.9.2 method=GET statusCode=200 path=/api/v1/accounts/verify_credentials msg="OK: wrote 734B"
```

The direct to login in the browser succeeds, the login itself succeeds with the message to confirm access for the app, and the redirect to Fedilab succeeds. Fedilab responds with "failed to get a token." There is no further message in the GTS logs related to this transaction, and no error in the logs. 

I don't see anything in the release notes about required changes to the configuration; here's mine. Notes: GTS is proxied behind Caddy, which is providing the certs. GTS is bound to a VPN address between the Caddy server and the GTS server -- it is not accessible directly.  The Caddy server is the trusted-proxy. The `gotosocial` directory is a symlink:

```
$ ls -ld gotosocial.d/gotosocial
lrwxrwxrwx 1 ser ser 20 Feb 11 19:44 gotosocial.d/gotosocial -> gotosocial-0.7.0-rc2
$ ps awwux | grep gotosocial | grep -v grep
ser      3800775  0.2  1.3 935860 108052 ?       Ssl  Feb11   5:01 /home/ser/gotosocial.d/gotosocial/gotosocial --config-path /home/ser/gotosocial.d/config.yaml server start
$ ls -lh gotosocial.d/data/sqlite.db
-rw-r--r-- 1 ser ser 156M Feb 13 09:08 gotosocial.d/data/sqlite.db
$ du -sh gotosocial.d/data
2.3G    gotosocial.d/data
$ cat gotosocial.d/gotosocial/config.yaml
host: "ap.ser1.net"
bind-address: "10.79.56.4"
port: 4865
db-type: "sqlite"
db-address: "/home/ser/gotosocial.d/data/sqlite.db"
storage-local-base-path: "/home/ser/gotosocial.d/data"
lets-encrypt-enabled: false
account-domain: "ser1.net"
trusted-proxies:
  - "10.79.56.7"
web-template-base-dir: "/home/ser/gotosocial.d/gotosocial/web/template/"
web-asset-base-dir: "/home/ser/gotosocial.d/gotosocial/web/assets/"
instance-expose-public-timeline: true
media-remote-cache-days: 24
accounts-registration-open: true
accounts-approval-required: true
accounts-reason-required: false
accounts-allow-custom-css: false
```

Incidentally, I "cleaned" all storage older than 10 days:

![2023-02-13-093713_1113x712_scrot](https://user-images.githubusercontent.com/60757196/218502843-961cf96c-f6a2-4726-8b06-d6eecde4dcbb.png)

with these server logs:

```
Feb 13 09:39:18 tethys.ser1.net gotosocial[3800775]: timestamp="13/02/2023 09:39:18.840" func=middleware.Logger.func1.1 level=INFO latency=925.796µs clientIP=198.54.135.138
userAgent="Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.0 Safari/605.1.15 Surf/2.1" method=POST statusCode=200 path=/api/v1/admin/media_cleanup msg="OK: wrote 26B"
Feb 13 09:39:18 tethys.ser1.net gotosocial[3800775]: timestamp="13/02/2023 09:39:18.841" func=media.(*manager).PruneAll.func1 level=INFO msg="pruned 0 unused local media"
Feb 13 09:39:20 tethys.ser1.net gotosocial[3800775]: timestamp="13/02/2023 09:39:20.593" func=media.(*manager).PruneAll.func1 level=INFO msg="pruned 19 unused remote media"
Feb 13 09:39:20 tethys.ser1.net gotosocial[3800775]: timestamp="13/02/2023 09:39:20.596" func=media.(*manager).PruneAll.func1 level=INFO msg="uncached 0 remote media older than 10 day(s)"
Feb 13 09:39:25 tethys.ser1.net gotosocial[3800775]: timestamp="13/02/2023 09:39:25.559" func=media.(*manager).PruneAll.func1 level=INFO msg="pruned 0 orphaned media"
Feb 13 09:39:25 tethys.ser1.net gotosocial[3800775]: timestamp="13/02/2023 09:39:25.677" func=media.(*manager).PruneAll.func1 level=INFO msg="cleaned storage"
```

and no change:

```
$ du -sh gotosocial.d/data
2.3G    gotosocial.d/data
```

**Comment by xxxserxxx** [comment]
On a related note, I tried to add a new user with tut, which #889 enabled and which had worked for me in <= 0.6.0. I assume 0.7.0 includes this, but I'm getting a 404 error page, instead:

```
$ tut --new-user
You will have to log in to your Mastodon instance to be able
to use Tut. The default protocol is https:// so you won't need
it. E.g. write fosstodon.org and press <Enter>.
--------------------------------------------------------------
Instance: ap.ser1.net
You need to authorize Tut to use your account. Your browser
should've opened. If not you can use the URL below.

https://ap.ser1.net/oauth/authorize?client_id=xxxxxxxxxxxxxxxxx&redirect_uri=urn%3Aietf%3Awg%3Aoauth%3A2.0%3Aoob&response_type=code&scope=read+write+follow

Authorization code:
```

A browser window is opened with the GTS login screen, which I enter credentials for; I then get:

![2023-02-13-091316_923x308_scrot](https://user-images.githubusercontent.com/60757196/218496683-d59b9a94-a88d-4515-8e8c-140a73f8d744.png)
and after clicking "Allow":
![2023-02-13-091330_917x294_scrot](https://user-images.githubusercontent.com/60757196/218496753-b53ec788-3e1d-4c27-9f4c-b539c25231e5.png)
I didn't cap the entire screen, for readability, but this is at the footer of all of the screens:
![2023-02-13-091348_459x101_scrot](https://user-images.githubusercontent.com/60757196/218496803-685e782f-7686-45e2-8878-dc43edf95f7e.png)

I've checked, and `web/template/oob.tmpl` exists.

**Comment by xxxserxxx** [comment]
I tried a couple of new things. First, I tried the new binary, 0.7.0. I re-iterate that the network configuration and proxy have not changed; the only thing that's changed since it last worked was the version of GTS. 

- Any logged-in clients were able to still log in.
- Able to log in through the web UI
- Unable to log in ("add account") with either tut or Fedilab

I first started by creating an export and a filesystem snapshot with GTS 0.7.0-rc2:

```bash
$ systemctl --user stop gotosocial
$ gotosocial-0.7.0-rc2 admin export --config-path config.yaml --path export.json
$ sudo btrfs subvolume snapshot -r / /snapshots/2023-02-11
```

## Attempt 1

1. Update the config, per the release instructions
   - Changed `odc-link-existing: true`; all other new attrs left to default
2. Start new GTS 0.7.0; let DB migration run
3. Try to log in with Fedilab

**Result:** The login sequence went as expected *until* the redirect back to the app, when it reported (as before): "The app failed to get a token"

## Attempt 2

1. Stop GTS 0.7.0
2. Move data directory out of the way (`storage-local-base-path`, `db-address` for sqlite)
3. Create a new empty directory to contain `storage-local-base-path` and `db-address`
4. Import previous export
   `gotosocial-0.7.0 admin import --config-path config.yaml --path export.json`
5. Import appeared to process with no errors
6. Start GTS 0.7.0
7. Try to log in with Fedilab

**Result:** The login sequence went as expected *until* the redirect back to the app, when it reported (as before): "The app failed to get a token"

GTS stdout logs of attempted login:

```
timestamp="17/02/2023 09:53:48.924" func=middleware.Logger.func1.1 level=INFO latency=285.706µs clientIP=AA.BB.CC.DD userAgent=okhttp/4.9.2 method=GET statusCode=200 path=/.well-known/nodeinfo msg="OK: wrote 114B"
timestamp="17/02/2023 09:53:49.356" func=middleware.Logger.func1.1 level=INFO latency=844.865µs clientIP=AA.BB.CC.DD userAgent=okhttp/4.9.2 method=GET statusCode=200 path=/nodeinfo/2.0 msg="OK: wrote 192B"
timestamp="17/02/2023 09:53:50.445" func=middleware.Logger.func1.1 level=INFO latency=1.207655ms clientIP=AA.BB.CC.DD userAgent=okhttp/4.9.2 method=POST statusCode=200 path=/api/v1/apps msg="OK: wrote 202B"
timestamp="17/02/2023 09:53:51.325" func=middleware.Logger.func1.1 level=INFO latency=787.026µs clientIP=AA.BB.CC.DD userAgent="Mozilla/5.0 (Android 13; Mobile; rv:109.0) Gecko/109.0 Firefox/109.0" method=GET statusCode=303 path=/oauth/authorize msg="See Other: wrote 64B"
timestamp="17/02/2023 09:53:51.740" func=middleware.Logger.func1.1 level=INFO latency=82.084µs clientIP=AA.BB.CC.DD userAgent="Mozilla/5.0 (Android 13; Mobile; rv:109.0) Gecko/109.0 Firefox/109.0" method=GET statusCode=304 path=/assets/dist/_colors.css msg="Not Modified: wrote 0B"
timestamp="17/02/2023 09:53:51.971" func=middleware.Logger.func1.1 level=INFO latency=51.857µs clientIP=AA.BB.CC.DD userAgent="Mozilla/5.0 (Android 13; Mobile; rv:109.0) Gecko/109.0 Firefox/109.0" method=GET statusCode=304 path=/assets/dist/base.css msg="Not Modified: wrote 0B"
timestamp="17/02/2023 09:53:51.972" func=middleware.Logger.func1.1 level=INFO latency=1.217192ms clientIP=AA.BB.CC.DD userAgent="Mozilla/5.0 (Android 13; Mobile; rv:109.0) Gecko/109.0 Firefox/109.0" method=GET statusCode=200 path=/assets/logo.png msg="OK: wrote 26.6kiB"
timestamp="17/02/2023 09:53:51.997" func=transport.(*transport).do level=INFO pubKeyID=https://ap.ser1.net/users/ser/main-key method=GET url=https://mastodon.social/.well-known/webfinger?resource=acct:The_Whore_of_Blahbylon@mastodon.social msg="performing request"
timestamp="17/02/2023 09:53:52.021" func=transport.(*transport).do level=INFO pubKeyID=https://ap.ser1.net/users/ser/main-key method=GET url=https://mastodon.social/users/The_Whore_of_Blahbylon msg="performing request"
timestamp="17/02/2023 09:53:52.072" func=middleware.Logger.func1.1 level=INFO latency=77.86817ms clientIP=2a01:4f8:171:a6::2 userAgent="http.rb/5.1.1 (Mastodon/4.1.0rc3; +https://mastodon.social/)" method=POST statusCode=200 path=/users/ser/inbox msg="OK: wrote 23B"
timestamp="17/02/2023 09:53:52.357" func=middleware.Logger.func1.1 level=INFO latency=115.576µs clientIP=AA.BB.CC.DD userAgent="Mozilla/5.0 (Android 13; Mobile; rv:109.0) Gecko/109.0 Firefox/109.0" method=GET statusCode=304 path=/assets/fonts/noto-sans-v27-latin-700.woff2 msg="Not Modified: wrote 0B"
timestamp="17/02/2023 09:53:52.362" func=middleware.Logger.func1.1 level=INFO latency=91.952µs clientIP=AA.BB.CC.DD userAgent="Mozilla/5.0 (Android 13; Mobile; rv:109.0) Gecko/109.0 Firefox/109.0" method=GET statusCode=304 path=/assets/fonts/noto-sans-v27-latin-regular.woff2 msg="Not Modified: wrote 0B"
timestamp="17/02/2023 09:54:16.108" func=middleware.Logger.func1.1 level=INFO latency=72.397601ms clientIP=AA.BB.CC.DD userAgent="Mozilla/5.0 (Android 13; Mobile; rv:109.0) Gecko/109.0 Firefox/109.0" method=POST statusCode=302 path=/auth/sign_in msg="Found: wrote 23B"
timestamp="17/02/2023 09:54:16.591" func=middleware.Logger.func1.1 level=INFO latency=2.301135ms clientIP=AA.BB.CC.DD userAgent="Mozilla/5.0 (Android 13; Mobile; rv:109.0) Gecko/109.0 Firefox/109.0" method=GET statusCode=200 path=/oauth/authorize msg="OK: wrote 852B"
timestamp="17/02/2023 09:54:16.999" func=middleware.Logger.func1.1 level=INFO latency=90.58µs clientIP=AA.BB.CC.DD userAgent="Mozilla/5.0 (Android 13; Mobile; rv:109.0) Gecko/109.0 Firefox/109.0" method=GET statusCode=304 path=/assets/dist/_colors.css msg="Not Modified: wrote 0B"
timestamp="17/02/2023 09:54:16.999" func=middleware.Logger.func1.1 level=INFO latency=46.007µs clientIP=AA.BB.CC.DD userAgent="Mozilla/5.0 (Android 13; Mobile; rv:109.0) Gecko/109.0 Firefox/109.0" method=GET statusCode=304 path=/assets/dist/base.css msg="Not Modified: wrote 0B"
timestamp="17/02/2023 09:54:17.016" func=middleware.Logger.func1.1 level=INFO latency=106.58µs clientIP=AA.BB.CC.DD userAgent="Mozilla/5.0 (Android 13; Mobile; rv:109.0) Gecko/109.0 Firefox/109.0" method=GET statusCode=304 path=/assets/logo.png msg="Not Modified: wrote 0B"
timestamp="17/02/2023 09:54:22.719" func=middleware.Logger.func1.1 level=INFO latency=1.432987ms clientIP=2600:3c03::f03c:93ff:fe42:5a14 userAgent="Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:106.0) Gecko/20100101 Firefox/106.0" method=GET statusCode=404 path=/api/v1/streaming/public/local msg="Not Found: wrote 1.68kiB"
timestamp="17/02/2023 09:54:23.533" func=middleware.Logger.func1.1 level=INFO latency=2.835568ms clientIP=74.208.135.95 userAgent="http.rb/5.1.1 (Mastodon/4.1.0; +https://universeodon.com/)" method=POST statusCode=200 path=/users/ser/inbox msg="OK: wrote 23B"
timestamp="17/02/2023 09:54:23.533" func=processing.(*processor).ProcessFromFederator level=INFO activityType=Create objectType=Note toAccount=ser iri=https://mastodon.social/users/argonaut/statuses/109880870003560515 msg="processing from federator"
timestamp="17/02/2023 09:54:23.541" func=transport.(*transport).do level=INFO pubKeyID=https://ap.ser1.net/users/ser/main-key method=GET url=https://mastodon.social/users/argonaut/statuses/109880870003560515 msg="performing request"
timestamp="17/02/2023 09:54:23.776" func=middleware.Logger.func1.1 level=INFO latency=209.202µs clientIP=2a01:4f8:171:b9b::2 userAgent="http.rb/5.1.1 (Mastodon/4.1.0rc3; +https://mastodon.social/)" method=GET statusCode=200 path=/users/ser/main-key msg="OK: wrote 742B"
timestamp="17/02/2023 09:54:24.058" func=transport.(*transport).do level=INFO pubKeyID=https://ap.ser1.net/users/ser/main-key method=GET url=https://mastodon.social/actor#main-key msg="performing request"
timestamp="17/02/2023 09:54:24.097" func=transport.(*transport).do level=INFO pubKeyID=https://ap.ser1.net/users/ser/main-key method=GET url=https://mastodon.social/actor msg="performing request"
timestamp="17/02/2023 09:54:24.106" func=transport.(*transport).do level=INFO pubKeyID=https://ap.ser1.net/users/ser/main-key method=GET url=https://mastodon.social/.well-known/webfinger?resource=acct:mastodon.social@mastodon.social msg="performing request"
timestamp="17/02/2023 09:54:24.115" func=middleware.Logger.func1.1 level=INFO latency=59.973883ms clientIP=2a01:4f8:171:b9b::2 userAgent="http.rb/5.1.1 (Mastodon/4.1.0rc3; +https://mastodon.social/)" method=GET statusCode=200 path=/users/ser msg="OK: wrote 928B"
timestamp="17/02/2023 09:54:25.667" func=transport.(*transport).do level=INFO pubKeyID=https://ap.ser1.net/users/ser/main-key method=GET url=https://mastodon.social/users/argonaut msg="performing request"
timestamp="17/02/2023 09:54:25.717" func=transport.(*transport).do level=INFO pubKeyID=https://ap.ser1.net/users/ser/main-key method=GET url=https://mastodon.social/.well-known/webfinger?resource=acct:argonaut@mastodon.social msg="performing request"
timestamp="17/02/2023 09:54:25.726" func=transport.(*transport).do level=INFO pubKeyID=https://ap.ser1.net/users/ser/main-key method=GET url=https://files.mastodon.social/accounts/avatars/000/413/662/original/75579ab44e12a6fa.png msg="performing request"
timestamp="17/02/2023 09:54:25.774" func=transport.(*transport).do level=INFO pubKeyID=https://ap.ser1.net/users/ser/main-key method=GET url=https://files.mastodon.social/accounts/headers/000/413/662/original/a02d2bf2088405f0.png msg="performing request"
timestamp="17/02/2023 09:54:45.125" func=middleware.Logger.func1.1 level=INFO latency=2.403568ms clientIP=AA.BB.CC.DD userAgent="Mozilla/5.0 (Android 13; Mobile; rv:109.0) Gecko/109.0 Firefox/109.0" method=POST statusCode=302 path=/oauth/authorize msg="Found: wrote 23B"
timestamp="17/02/2023 09:54:46.050" func=middleware.Logger.func1.1 level=INFO latency=1.166718ms clientIP=AA.BB.CC.DD userAgent=okhttp/4.9.2 method=POST statusCode=200 path=/oauth/token msg="OK: wrote 160B"
timestamp="17/02/2023 09:54:46.529" func=transport.(*transport).do level=INFO pubKeyID=https://ap.ser1.net/users/ser/main-key method=GET url=https://botsin.space/users/primes#main-key msg="performing request"
timestamp="17/02/2023 09:54:46.838" func=middleware.Logger.func1.1 level=INFO latency=3.31114ms clientIP=AA.BB.CC.DD userAgent=okhttp/4.9.2 method=GET statusCode=200 path=/api/v1/accounts/verify_credentials msg="OK: wrote 649B"
timestamp="17/02/2023 09:54:47.297" func=transport.(*transport).do level=INFO pubKeyID=https://ap.ser1.net/users/ser/main-key method=GET url=https://botsin.space/users/primes msg="performing request"
timestamp="17/02/2023 09:54:47.398" func=transport.(*transport).do level=INFO pubKeyID=https://ap.ser1.net/users/ser/main-key method=GET url=https://botsin.space/.well-known/webfinger?resource=acct:primes@botsin.space msg="performing request"
timestamp="17/02/2023 09:54:47.500" func=transport.(*transport).do level=INFO pubKeyID=https://ap.ser1.net/users/ser/main-key method=GET url=https://files.botsin.space/accounts/avatars/000/023/146/original/b87d1f05456a00f4.png msg="performing request"
timestamp="17/02/2023 09:54:47.894" func=transport.(*transport).do level=INFO pubKeyID=https://ap.ser1.net/users/ser/main-key method=GET url=https://files.botsin.space/accounts/headers/000/023/146/original/9c919bfb24dc06a8.png msg="performing request"
timestamp="17/02/2023 09:54:48.268" func=dereferencing.(*deref).enrichAccount level=ERROR msg="error fetching remote header for account https://botsin.space/users/primes: error reading incoming media: unexpected EOF"
timestamp="17/02/2023 09:54:48.273" func=middleware.Logger.func1.1 level=INFO latency=1.747584515s clientIP=147.182.217.82 userAgent="http.rb/5.1.1 (Mastodon/4.1.0; +https://botsin.space/)" method=POST statusCode=200 path=/users/ser/inbox msg="OK: wrote 23B"
timestamp="17/02/2023 09:54:48.274" func=processing.(*processor).ProcessFromFederator level=INFO activityType=Create objectType=Note toAccount=ser msg="processing from federator"
```

With as clean of a DB as I can get (while retaining my account), I'm still unable to log in with Fedilab with the new GTS builds.

The caddy reverse proxy is trivial:

```
ap.ser1.net {
   log
   reverse_proxy http://LOCALVPNIP:PORT
}
```

and it certainly works, as I can browse to GTS via the (public) web and successfully log in.
Is there anything else I can try, barring wiping everything, recreating my account, and re-subscribing to everyone?

**Comment by xxxserxxx** [comment]
I created a completely new user and tried Fedilab and tut, with the same results.

However, I just saw #1518, which lead me to investigate the URL on the 404 page and found the token; this was not obvious to me because I have my URL bar set to invisible. After copying the code out of the URL on the 404 web page, I was able to log in with tut.

Fedilab is still a failure to log in. Are these at all related? Can someone else confirm that -- logging out with Fedilab -- they're able to log back in?
